examples/step-50/doc/intro.dox

```diff
@@ -134,3 +191,64 @@ due to the asynchronous work "covering up" the imbalance (which assumes synchron
 However, for most realistic adaptive meshes the expectation is that this asynchronous work will
 only cover up a very small portion of the imbalance and the efficiency model will describe the
 slowdown very well.
+
+
+<h3>Workload imbalance for algebraic multigrid methods</h3>
+
+The considerations above show that one has to expect certain limits on
+the scalability of geometric multigrid methods because even in cases
```

Comment:
```suggestion
the scalability of the geometric multigrid algorithm as it is implemented in deal.II because even in cases
```
The imperfection is not a given and could be fixed by a more clever strategy for distributing the level cells.